modules/mobile/spec/request/letters_request_spec.rb

```diff
@@ -264,20 +265,11 @@
       it 'matches the letters schema' do
```

Comment:
Looks like this description was wrong to begin with. Do you mind changing it to "returns bad request"?

Reply:
Sure thing